packages/editor-server/src/server/pandoc.ts

```diff
@@ -55,6 +55,13 @@ import { EditorServerOptions } from './server';
 
 export function pandocServer(options: EditorServerOptions) : PandocServer {
 
+  // work around change in pandoc 3.2+: https://github.com/jgm/pandoc/issues/9677 by
+  // using a custom pandoc writer
+  function substituteCustomMarkdownWriter(format: string): string {
+    const customWriterScript = path.join(options.pandoc.resourcesDir, 'md-writer.lua')
+    return format.replace(/^markdown(?=[-+]|$)/, customWriterScript);
```

Comment:
This will eat all extensions that might be passed as `format`, right? Is that a problem? I'm not sure about the context around `astToMarkdown`, but it's probably worth checking.

Reply:
No, I preserve everything after `markdown`, so if the format string comes in as:

`markdown+foo-bar` that gets replaced by `/path/to/md-writer.lua+foo-bar` and so on.

Reply:
Hmm, let me double check that bold statement.

Reply:
Double-checked.

<img width="457" alt="regex-test" src="https://github.com/user-attachments/assets/2102e675-ef76-4040-ad2d-eeda9408770b">

Reply:
It's been 0 days since a regexp misreading in this factory

Reply:
Docs on passing extensions to custom writers: https://github.com/jgm/pandoc/blob/main/doc/custom-writers.md#format-extensions